### DISCUSSION TITLE
My AI chatbot says that I've used up my message limit.

### DISCUSSION THREAD

**Discussion by mrodiculous** [AntlerVC/firetable#1461] [Support / Q&A]
### Discussed in https://github.com/rowyio/rowy/discussions/1425

<div type='discussions-op-text'>

<sup>Originally posted by **mrodiculous** September 30, 2023</sup>
Hi, I have a pro paid account and I keep getting the message, "Looks like you have used up your message limit! But don't worry, you can keep the conversation going by creating your own WhatsApp bot for free using your OpenAI account. Go to aibot.how to get started. It's super easy, without using any code, and your data stays secure on your own OpenAI account. Give it a try!"  What is wrong?</div>

**Comment by harinij** [comment]
Hi @mrodiculous - it seems you are using the whatsapp bot template and this feature of the template which is limited in the number of message it can receive. You can look into the message derivative column and remove this check for limits in place. Join our [discord](https://buildship.com/discord) for getting wider community support if you are still looking for a solution:
